README.md

```diff
@@ -1,22 +1,23 @@
 # X Components Archetype
 
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Created by](https://img.shields.io/badge/Created%20by-Empathy.co-green)](https://www.empathy.co)
+
 Welcome to the X Components Archetype
 
 ## What is this repository for?
 
-The purpose of this repository is to be the starting point to a customer setup using the X
-Components. Please take a look at the [documentation](./docs/index.md).
+The purpose of this repository is to be the starting point to a customer setup using the X Components. Please take a look at
+the [documentation](./docs/index.md).
```

Comment:
should we update the documentation? It is referring bitbucket

Reply:
We definitely should discuss about changes in the documentation. How is this project going to be used? How are clients setups/projects going to be handled/managed? Are they going to be public in github? Cause I don't think so... 

So, in the end, I think we should have a team discussion about this topic.

Reply:
Yes, we should have that discussion. The client repos are going to be in Github, but private of course. We should treat this repo as the default/preferred/mostloved/startingpoint search experience to be forked/duplicated to start an easy instrumentation.